src/renderer/components/+workloads-daemonsets/daemonset-details.tsx

```diff
@@ -58,6 +62,17 @@ export class DaemonSetDetails extends React.Component<Props> {
     this.metrics = null;
   });
 
+  @disposeOnUnmount
+  revisionLoader = autorun(async () => {
+    const { object: daemonSet } = this.props;
+    const revisionHistory = await controllerRevisionApi.getRevisions({
+      name: daemonSet.getName(),
+      namespace: daemonSet.getNs()
+    }) as ControllerRevision[];
+
+    this.revisionHistory = observable.array(revisionHistory);
```

Comment:
This is redundent, please change to `this.revisionHistory.replace(revisionHistory)`